### PR TITLE
[NG] add a golden file test

### PR DESCRIPTION
**Pull request by gnomeontherun**
This is a new type of test that captures our public API as part of our build process and tests against the previous version to check if any changes were found. This will help us identify when changes in our public api happen.

### CODE REVIEW

**Comment by gnomeontherun** [comment]
This is a run with an outdated golden file to see it failing, now I'll update the golden file for latest changes. (This version was built before the clrIfOpen directives were made public)

File: src/clr-angular/index.ts [hunk 1]
```diff
@@ -18,6 +18,3 @@ export * from './modal/index';
 export * from './popover/index';
 export * from './utils/index';
 export * from './wizard/index';
-
-// Below are exported for internal use only and may change without notice
-export { FocusTrapTracker as ÇlrFocusTrapTracker } from './utils/focus-trap/focus-trap-tracker.service';
```
[the remark this answers is not in the document]
Comment (youdz) [reply src/clr-angular/index.ts]: The website needs access to the `FocusTrapTracker` to override it, so we can't remove the actual export. Whether it's part of the golden files or not doesn't really matter, but it has to be exported.

[the remark this answers is not in the document]
Comment (gnomeontherun) [reply src/clr-angular/index.ts]: The tool was throwing an error about not supporting aliases for the golden file. Turns out you can also explicitly ignore certain patterns so I ignored the focustraptracker to get it to work correctly. So this line has been reverted.

**Comment by gnomeontherun** [comment]
@youdz please double check changes

**Pull request merged**